pandas/tseries/offsets.py

```diff
@@ -1142,9 +1173,9 @@ class SemiMonthEnd(SemiMonthOffset):
 
     Parameters
     ----------
-    n : int
-    normalize : bool, default False
-    day_of_month : int, {1, 3,...,27}, default 15
+        n : int
+        normalize : bool, default False
+        day_of_month : int, {1, 3,...,27}, default 15
```

Comment:
I don't think this indentation is correct, is it?

Reply:
No it isn't. My bad.